Locker/Internal/LockerHelpers.swift

```diff
@@ -0,0 +1,226 @@
+        LockerHelpers.deviceCode == "x86_64"
+    }
+
+    // MARK: - Private properties
+
+    static private var currentLAPolicyDomainState: Data? {
+        let context = LAContext()
+        context.canEvaluatePolicy(.deviceOwnerAuthenticationWithBiometrics, error: nil)
+        return context.evaluatedPolicyDomainState
+    }
+
+    static private var savedLAPolicyDomainState: Data? {
+        UserDefaults.standard.object(forKey: LockerHelpers.keyLAPolicyDomainState) as? Data
+    }
+
+    private static let bundleIdentifier = Bundle.main.bundleIdentifier ?? ""
+}
+
+// MARK: - Public extension
+
+extension LockerHelpers {
+
+    // MARK: - User defaults keys help methods
+
+    static func keyKeychainAccountNameForUniqueIdentifier(_ uniqueIdentifier: String) -> String {
+        let keychainAccountName = "\(LockerHelpers.bundleIdentifier)_KeychainAccount"
+        return "\(keychainAccountName)_\(uniqueIdentifier)"
+    }
+
+    static func keyDidAskToUseBiometricsIDForUniqueIdentifier(_ uniqueIdentifier: String) -> String {
+        let userDefaultsDidAskToUseBiometricsID = "\(LockerHelpers.bundleIdentifier)_UserDefaultsDidAskToUseTouchID"
+        return "\(userDefaultsDidAskToUseBiometricsID)_\(uniqueIdentifier)"
+    }
+
+    static func keyBiometricsIDActivatedForUniqueIdentifier(_ uniqueIdentifier: String) -> String {
+        let userDefaultsKeyBiometricsIDActivated = "\(LockerHelpers.bundleIdentifier)_UserDefaultsKeyTouchIDActivated"
+        return "\(userDefaultsKeyBiometricsIDActivated)_\(uniqueIdentifier)"
+    }
+
+    static func keyShouldAddSecretToKeychainOnNextLoginForUniqueIdentifier(_ uniqueIdentifier: String) -> String {
+        let userDefaultsShouldAddSecretToKeychainOnNextLogin = "\(LockerHelpers.bundleIdentifier)_UserDefaultsShouldAddPasscodeToKeychainOnNextLogin"
+        return "\(userDefaultsShouldAddSecretToKeychainOnNextLogin)_\(uniqueIdentifier)"
+    }
+
+    // MARK: - Biometric helpers
+
+    static func storeCurrentLAPolicyDomainState() {
+        let newDomainState = LockerHelpers.currentLAPolicyDomainState
+        LockerHelpers.setLAPolicyDomainState(with: newDomainState)
+    }
+
+    static var keyKeychainServiceName: String {
+        return "\(LockerHelpers.bundleIdentifier)_KeychainService"
+    }
+}
+
+// MARK: - Private extension
+
+private extension LockerHelpers {
+
+    static func checkIfBiometricsSettingsChanged() -> Bool {
+        let oldDomainState = LockerHelpers.savedLAPolicyDomainState
+        let newDomainState = LockerHelpers.currentLAPolicyDomainState
+
+        // Check for domain state changes
+        // For deactivated biometrics, LAContext in validation will return nil
+        // storing that nil and comparing it to nil will result as `isEqual` NO
+        // even data is not actually changed.
+        let biometricsDeactivated = oldDomainState == nil || newDomainState == nil
+        let biometricSettingsDidChange = oldDomainState?.elementsEqual(newDomainState!) ?? false
```

Comment:
Avoid force unwrap, instead, use `guard let` and update the documentation as it reflects only ObjC implementation.

I would suggest creating something like this:
```swift
        guard let oldDomainState = LockerHelpers.savedLAPolicyDomainState,
              let newDomainState = LockerHelpers.currentLAPolicyDomainState,
              oldDomainState.elementsEqual(newDomainState)
        else {
            LockerHelpers.setLAPolicyDomainState(with: LockerHelpers.currentLAPolicyDomainState)
            return true
        }
        return false
```